packages/liveblocks-react-ui/src/components/Composer.tsx

```diff
@@ -349,6 +374,97 @@ function ComposerMentionSuggestions({
   ) : null;
 }
 
+function MarkToggle({ mark, shortcut, children, ...props }: MarkToggleProps) {
+  const $ = useOverrides();
+  const label = useMemo(() => {
+    return $.COMPOSER_TOGGLE_MARK(mark);
+  }, [$, mark]);
```

Comment:
I do have a tendency to overuse `useMemo`/`useCallback` but here the `COMPOSER_TOGGLE_MARK` overrides can be set by users so it might be unexpected to run it on every render. I don't think anyone would put super expensive logic in a localization function but still I feel safer controlling its number of invokations